### PR TITLE
Debian packaging improvements

### PR DESCRIPTION
**Pull request by joshuaboniface**
Multiple improvements to the Debian packaging, including:

1. Renaming everything to `jellyfin`.
2. Fix up the initscript/unit file/upstart job to use proper `dotnet` startup.
3. Replace hardcoded `/var/lib/emby` path in `MediaBrowser.Server.Mono/app.config`.

### CODE REVIEW

**Comment by joshuaboniface** [comment]
This builds a Debian binary package with a working service for me.

```
$ systemctl status jellyfin
● jellyfin.service - Jellyfin Media Server
   Loaded: loaded (/lib/systemd/system/jellyfin.service; enabled; vendor preset: enabled)
   Active: active (running) since Sun 2018-12-09 17:38:25 EST; 8min ago
 Main PID: 5588 (dotnet)
   CGroup: /system.slice/jellyfin.service
           └─5588 /usr/bin/dotnet /usr/lib/jellyfin/bin/EmbyServer.dll
```

File: debian/bin/jellyfin [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      usermod -a -G jellyfin $JELLYFIN_USER
+    else
+      echo "WARNING: The runas user is not part of jellyfin group and you don't have enough privileges to add it. The restart button in the GUI will probably fail."
+      echo "To solve this start the jellyfin service using the startup scripts (systemd/sysv) or"
+      echo "add the runas user ($JELLYFIN_USER) to the jellyfin group manually and restart Emby."
+    fi
+  fi
+fi
+
+# Data directory where Emby database, cache and logs are stored
```
Comment (JustAMan) [inline debian/bin/jellyfin]: This comment needs fixing :-)

File: debian/jellyfin.init [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Description:       Runs Emby Server
+### END INIT INFO
+
+# Carry out specific functions when asked to by the system
+pid=`ps -fA|grep dotnet|grep EmbyServer|awk '{print $2}'| tr -d '\n'`
+
+case "$1" in
+  start)
+        if [ "$pid" == "" ]; then
+                echo "Starting Emby..."
```
Comment (JustAMan) [inline debian/jellyfin.init]: And those echo lines also need a naming fix

**Comment by joshuaboniface** [comment]
Yea, there is a ton wrong with this branch, I was too hasty in self-merging - I've reverted it to do some more testing.

**Pull request merged**